proto/osmosis/streamswap/v1/state.proto

```diff
@@ -41,11 +40,13 @@ message Sale {
   // Last round of the Sale;
   int64 end_round = 9;
```

Comment:
proto field numbers need to change? (or is this intened?

Reply:
What do you mean?